modules/sno-du-prerequisites.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/sno-du-deploying-clusters-on-single-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="sno-du-prerequisites_{context}"]
+= Prerequisites
+
+* A namespace for the cluster must exist for the unique
+configuration for the single node being installed.
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+    labels:
+        name: sno-du-1 <1>
+    name: sno-du-1
+----
+<1> The namespace selected for the particular cluster. You need one per system that is installed.
+
+* You must propagate the secrets for the cluster, as shown in the following command:
```

Comment:
You must propagate the secrets for the cluster to this namespace, including the bare-metal host credentials and OCP installation pull-secret. The pull secret can be propagated as show in the following command:

Reply:
changed